src/components/Categorical.tsx

```diff
@@ -54,29 +44,29 @@ function Categorical({
   onToggleMissingValue,
   onUpdateLevelTerm,
 }: CategoricalProps) {
-  const { getAssessmentToolConfig } = useDataStore();
+  const { isMultiColumnMeasureStandardizedVariable: isMultiMeasureColumnStandardizedVariable } =
+    useDataStore();
 
   const showStandardizedTerm =
-    standardizedVariable &&
-    standardizedVariable?.identifier !== getAssessmentToolConfig().identifier;
+    standardizedVariable && !isMultiMeasureColumnStandardizedVariable(standardizedVariable);
+  const [options, setOptions] = useState<StandardizedVariable[]>([]);
 
-  const getTermOptions = () => {
-    if (standardizedVariable?.identifier === 'nb:Diagnosis') {
-      return diagnosisTerms;
-    }
-    if (standardizedVariable?.identifier === 'nb:Sex') {
-      return sexTerms;
-    }
-    return [];
-  };
+  const { getTermOptions } = useDataStore();
 
-  const termOptions = getTermOptions();
+  useEffect(() => {
+    if (standardizedVariable) {
+      const fetchOptions = async () => {
+        try {
+          const result = await getTermOptions(standardizedVariable);
+          setOptions(result);
+        } catch (error) {
+          // TODO: show a notif error
+        }
+      };
```

Comment:
similar question to the multiColumn component: why is this an async operation? Never mind the fact that the source files are local. Why hasn't the term loading already happened at this point and I am just getting the terms from the store?